readme-vars.yml

```diff
@@ -67,6 +67,7 @@ app_setup_block: |
 
 # changelog
 changelogs:
+  - { date: "27.05.24:", desc: "Rebase to Alpine 3.20."}
```

Comment:
I think its worth noting that the default changes, and will produce warnings if not tended to. 
We could at least try to get less support questions.

Reply:
Probably something we need to standardise across everything using the nginx base